project/ws/app/src/lib/routes/home/components/filter/filter.component.ts

```diff
@@ -51,41 +52,46 @@ export class FilterComponent implements OnInit, AfterContentChecked {
   }
 
   ngOnInit() {
+    this.designationList = this.filterFacetsData.designation
```

Comment:
Please do check if this.filterFacetsData exists then do assign

Reply:
done